infra/modules/azure_app_service_plan_autoscaler/variables.tf

```diff
@@ -105,23 +105,69 @@ variable "scale_metrics" {
       lower_threshold = optional(number, 20)
       increase_by     = optional(number, 1)
       decrease_by     = optional(number, 1)
-      }), {
-      upper_threshold = 80
-      lower_threshold = 20
-      increase_by     = 1
-      decrease_by     = 1
-    })
+    }), {})
     memory = optional(object({
       upper_threshold = optional(number, 70)
       lower_threshold = optional(number, 20)
       increase_by     = optional(number, 1)
       decrease_by     = optional(number, 1)
-      }), {
-      upper_threshold = 70
-      lower_threshold = 20
-      increase_by     = 1
-      decrease_by     = 1
-    })
+    }), {})
+    cooldown = optional(object({
+      requests_rule = optional(object({
```

Comment:
Wouldn't be better to move this definition inside the existing one `requests`? It would end up with
```tf
    requests = optional(object({
      upper_threshold = number
      lower_threshold = number
      increase_by     = number
      decrease_by     = number
      cooldown_increase = number
      cooldown_decrease = number
      window = number
      aggregation = number
```

which looks smoother and less verbose

Reply:
`statistic`, `time_aggregation` and `time_window` are everytime the same for increase and decrease cases?